metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/configs/HiveConnectorFastServiceConfig.java

```diff
@@ -72,7 +71,7 @@ public HiveConnectorPartitionService fastHivePartitionService(
         final HiveConnectorInfoConverter hiveMetacatConverter,
         final ThreadServiceManager threadServiceManager,
         final ConnectorContext connectorContext,
-        @Qualifier("hiveDataSource") final DataSource dataSource,
+        @Qualifier("hiveJdbcTemplate") final JdbcTemplate hiveJdbcTemplate,
```

Comment:
This is the correct place for the `@Qualifier` the other one is redundant noise and ties the class to spring unnecessarily